docs/core/tools/csproj.md

```diff
@@ -69,6 +69,13 @@ This change does not modify the main mechanics of other includes. However, if yo
 ### Recommendation
 With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
```

Comment:
We've had an on-going, occasional background discussion about fixing other little probs as we go. Yes, it's not part of this ... it just looks bad ... these little grammatical problems. Sure, I'm cool either way. We can nab this on another PR if you like.